internal/wire/records_test.go

```diff
@@ -51,6 +53,15 @@ func loadRecords(recordsPath string) ([]testCase, error) {
 		return nil, err
 	}
 
+	// Select random N number of files from an array of files
+	N := 1000
+	if len(recordFiles) > N {
+		r := rand.New(rand.NewSource(time.Now().UnixNano()))
+		offset := r.Intn(len(recordFiles) - N)
+		limit := offset + N
+		recordFiles = recordFiles[offset:limit]
+	}
```

Comment:
I think we could simplify that with https://pkg.go.dev/math/rand#Perm and use returned slice (up to 1000) as a list of indexes

Reply:
but then we will always get the first 1000 files
It is acceptable?